### PR TITLE
 Build Clubs Curriculum page

### LINKED ISSUE #42: Build Clubs Curriculum page
### RACI
* Phase: Build
* Owner: @hannahkane 
* Decision Maker: @hannahkane 
* Design Lead: NA
* Dev Lead: 
* Quality: @davidascher 

Design assets will come from #106.

### Requirements
* Responsive
* Works in all supported browser versions
* Does NOT need to be dynamic. The curriculum titles/images/blurbs can be static for v1.
* Does NOT need to be l10n'ed for v1. Any translated curriculum can be linked to from the github pages themselves.

Tasks:
 - [ ] Design review
 - [ ] Content review

### PR DESCRIPTION
**Pull request by gesa**
Fixes #42

### CODE REVIEW

**Comment by ScottDowne** [comment]
I cases where there are a lot of 2x and 1x images, I tried organizing them in a 1x and 2x dir. Not sure if it made any diff though. I would also have an svg dir.

File: less/components/sidebar.less [hunk 1]
```diff
@@ -19,7 +19,7 @@ html.no-js .sidebar {
   &.teach-like-mozilla .sidebar {
     .sidebarColorizer(@tealSidebarColor);
   }
-  &.clubs .sidebar {
+  &.clubs .sidebar, &.clubs-curriculum-page .sidebar {
```
Comment (ScottDowne) [inline less/components/sidebar.less]: Interesting, I worked on a sub page, and I was given an off colour version of the parent page's colour. Maybe we can request a darker orange for clubs-curriculum?

Comment (gesa) [inline less/components/sidebar.less]: Direction @vazquez ? should I just do `darken(@orangeSidebarColor, 10%)` or something similar?

File: pages/clubs-curriculum.jsx [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+  render: function () {
+    return (
+        <section className="row web-lit-basics">
+          <div className="col-sm-12">
+            <h2>Web Literacy Basics</h2>
+            <p>
+              Learners get familiar with reading, writing and participating on the web in this six-part module.
+              Discover the foundations of the web through production and collaboration. The learning objectives
+              underpinning each activity are informed by
+              Mozilla&rsquo;s <a href="https://webmaker.org/en-US/resources">Web Literacy Map</a>
```
Comment (ScottDowne) [inline pages/clubs-curriculum.jsx]: I would double check with someone else that we don't want this linked to learn-like-mozilla/web-literacy instead

File: less/components/sidebar.less (hunk 1 above)
Comment (vazquez) [inline less/components/sidebar.less]: We would want to lighten. `lighten(@orangeSidebarColor, 30%)`;

**Comment by ScottDowne** [comment]
One last question, is it intended that these curriculum don't have links to anything?

File: pages/clubs-curriculum.jsx (hunk 2 above)
Comment (gesa) [inline pages/clubs-curriculum.jsx]: Confession: i have zero recollection creating this link.

Comment (gesa) [inline pages/clubs-curriculum.jsx]: Also @hannahkane if you could weigh in here?

**Comment by gesa** [comment]
@ScottDowne that is a very good point.

**Comment by gesa** [comment]
@hannahkane question again. Should these link to anything?

if so, can i get a list?

File: less/components/sidebar.less (hunk 1 above)
Comment (ScottDowne) [inline less/components/sidebar.less]: We need to make a new variable for this colour.

To me it looks kinda like a `@radicalPumpkinOrangeSidebarColor`

**Comment by hannahkane** [comment]
The Web Lit Map link should go to /activities/web-literacy (@iamjessklein to confirm)

And, yes, the curriculum sections should link!

Kraken: http://mozilla.github.io/webmaker-curriculum/WebLiteracyBasics-I/session01-kraken.html
Ping Kong: http://mozilla.github.io/webmaker-curriculum/WebLiteracyBasics-I/session01-pingkong.html
Hack the News: http://mozilla.github.io/webmaker-curriculum/WebLiteracyBasics-I/session02-hackthenews.html
HTML Puzzle Boxes: http://mozilla.github.io/webmaker-curriculum/WebLiteracyBasics-I/session02-puzzlebox.html
Web Chef: http://mozilla.github.io/webmaker-curriculum/WebLiteracyBasics-I/session03-chef.html
Final project: http://mozilla.github.io/webmaker-curriculum/WebLiteracyBasics-I/session03-storyofus.html

**Comment by gesa** [comment]
Okay, so. All those things have been fixed/added with the exception of the internal link to weblit which is currently a fixme/TODO because weblit isn't merged yet.

To watch out for: it *looks* like there are a lot of changes to `sidebar.less`, but really there aren't—i just un-nested some things that didn't need to be nested. It's still beautiful. (also I think I fixed that sidebar wrapping regression that @toolness had noticed so there's that)

**Comment by gesa** [comment]
(this build failed because i deleted the branch too quick)

**Comment by gesa** [comment]
(it's not really broke)

**Pull request merged**